.github/workflows/test.yml

```diff
@@ -33,17 +32,12 @@ jobs:
       - name: Build
         run: npm run build
 
-      - name: Run jest tests
```

Comment:
Coverage is slow. We may want to revert these changes, and simply `matrix.node == 10` to `matrix.node == 14` in the conditions.